public/data/Akshat-Mishra101.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Akshat Mishra",
+  "type": "personal",
+  "bio": "Software Engineer",
+  "avatar": "https://github.com/Akshat-Mishra101.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Akshat-Mishra101",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/@0xakshat",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@_akshat",
+      "icon": "medium"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

Comment:
I recommend capitalizing the `i` here. What do you think about this? 🙂
```suggestion
      "name": "Follow me on LinkedIn",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
Sure, let me fix this right away :)